lib/cosmos/dart/lib/dart_decom_query.rb

```diff
@@ -168,6 +168,8 @@ def query(request)
   # @param is_tlm true or false
   # @return [Array<String>] Array of item names
   def item_names(target_name, packet_name, is_tlm = true)
+    Cosmos::Logger.info("#{time.formatted}: item_names")
```

Comment:
It appears this is only being called upon initialization? Don't want to fill up the log with stuff ...

Reply:
item_names is used as part of the meta selection dialog.  Very infrequent.